events/events-team/content/sig-f2f.md

```diff
@@ -0,0 +1,19 @@
+# SIG Face-to-Face Handbook
```

Comment:
I think this content is a duplicate of the [SIG events role document](https://github.com/kubernetes/community/blob/master/events/events-team/sig-contrib-events.md) already in GitHub

Reply:
that wasn't merged when I wrote this stuff.  I expect the answer is to copy your, more complete document, over the sig-* documents in this PR after the PR merges.

Reply:
Stub definitions removed, sig-contrib-events.md moved to the content folder.